src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -154,26 +145,16 @@ def make_values(p: XMLProperty) -> list[dict[str, Any]]:
         # To frame the json-ld correctly, we need one property used in the graph. It does not matter which.
         last_prop_name = None
 
-        str_value_to_serialiser_mapper = {
-            "color": (SerialiseColor, StringTransformer()),
-            "date": (SerialiseDate, DateTransformer()),
-            "decimal": (SerialiseDecimal, DecimalTransformer()),
-            "geometry": (SerialiseGeometry, GeometryTransformer()),
-            "geoname": (SerialiseGeoname, StringTransformer()),
-            "time": (SerialiseTime, StringTransformer()),
-            "uri": (SerialiseURI, StringTransformer()),
-        }
```

Comment:
I moved this away, because it is not dependent of its encapsulating function. It always stays the same, so you don't have to build it again and again.